src/wfs/src/main/java/org/geoserver/wfs/InsertElementHandler.java

```diff
@@ -271,7 +271,7 @@ public Class getElementClass() {
                 typeNames.add(new QName(namespaceURI, name));
             }
         } else {
-            LOGGER.finer("Insert was empty - does not need a FeatuerSoruce");
+            LOGGER.finer("Insert was empty - does not need a FeatureSource");
         }
```

Comment:
I do not see a fix here - only changes to logging, and test cases confirming the other commit worked.